ReactWindows/ReactNative.sln

```diff
@@ -145,6 +147,36 @@ Global
 		{5AC8108B-1C39-4C4A-8653-DBBE4ECCE691}.ReleaseBundle|x86.ActiveCfg = Release|x86
 		{5AC8108B-1C39-4C4A-8653-DBBE4ECCE691}.ReleaseBundle|x86.Build.0 = Release|x86
 		{5AC8108B-1C39-4C4A-8653-DBBE4ECCE691}.ReleaseBundle|x86.Deploy.0 = Release|x86
+		{4B72C796-16D5-4E3A-81C0-3E36F531E578}.Debug|Any CPU.ActiveCfg = Debug|Win32
```

Comment:
Just capturing this.  In some projects, Any CPU points to x64 and in other projects it points to x86/Win32